test/test-server/utils.go

```diff
@@ -349,6 +351,37 @@ func buildIamInvitation(id, email, userId, status string) iamv2.IamV2Invitation
 	}
 }
 
+func buildIamGroupMapping(id, name, description, filter string) ssov2.IamV2SsoGroupMapping {
```

Comment:
Does this get used in more than one file in this package? If not, there's no need to put it in a different file than where it get used. (Results in needing to jump around more while reading code, and files with generic names like utils.go often have the tendency to get really big... like hundreds of lines)